carbon-p2-plugin/src/main/java/org/wso2/maven/p2/RepositoryGenMojo.java

```diff
@@ -291,8 +292,9 @@ private void copyBundleArtifacts()throws MojoExecutionException {
                 .hasNext();) {
             BundleArtifact bundleArtifact = (BundleArtifact) iterator.next();
             try {
-            	File file = bundleArtifact.getArtifact().getFile();
-                FileManagementUtil.copy(file, new File(pluginsDir,file.getName()));
+            	//This will be fixed in a different jira
```

Comment:
If this logic is not used anymore, I believe, this try-catch block should be removed altogether.